src/main/java/org/broadinstitute/hellbender/utils/read/markduplicates/GATKDuplicationMetrics.java

```diff
@@ -0,0 +1,66 @@
+package org.broadinstitute.hellbender.utils.read.markduplicates;
+
+import htsjdk.samtools.SAMRecord;
+import htsjdk.samtools.metrics.MetricBase;
+import htsjdk.samtools.util.Histogram;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.read.ReadUtils;
+import picard.sam.DuplicationMetrics;
+
+import java.io.Serializable;
+
+/**
+ * Metrics that are calculated during the process of marking duplicates
+ * within a stream of SAMRecords.
+ */
+@SuppressWarnings("serial")
```

Comment:
Instead of suppressing the serial warning we should add the serialVersionUID.  We've had problems in the past due to not including it.

Reply:
done